docs/platforms/python/migration/1.x-to-2.x.mdx

```diff
@@ -0,0 +1,103 @@
+---
+title: Migrate from 1.x to 2.x
+sidebar_order: 8998
+description: "Learn about migrating from sentry-python 1.x to 2.x"
+---
+
+<Note>
+
+There is no stable 2.0 release of the Sentry Python SDK yet. An alpha version (`2.0.0a1`) is available for testing. If you decide to give it a try, please [let us know](https://github.com/getsentry/sentry-python/issues) if you run into any issues!
+
+</Note>
+
+
+If you want to move to version 2.x of the `sentry-python` SDK, we provide a short guide here of the most common patterns.
+
+While the top level API stays the same for the most part, there's a number of deprecations and breaking changes in 2.0. Here's a selection that we think might affect the most people; for the full list of changes check out the [detailed migration guide in the repository](https://github.com/getsentry/sentry-python/blob/sentry-sdk-2.0/MIGRATION_GUIDE.md).
+
+## Configuration options
+
+The `profiles_sample_rate` and `profiler_mode` options are not experimental anymore and can be used directly. Setting them via `_experiments` is deprecated and will be removed in the future.
+
+The deprecated `with_locals` and `request_bodies` options have been removed in favor of their new counterparts: `include_local_variables` and `max_request_body_size`, respectively.
+
+**Old**:
+
+<SignInNote />
+
+```python
+import sentry_sdk
+
+sentry_sdk.init(
+    dsn="___PUBLIC_DSN___",
+    with_locals=False,
+    request_bodies="always",
+    _experiments={
+        "profiles_sample_rate": 1.0,
+        "profiler_mode": "thread",
+    },
+)
+```
+
+**New**:
+
+<SignInNote />
+
+```python
+import sentry_sdk
+
+sentry_sdk.init(
+    dsn="___PUBLIC_DSN___",
+    include_local_variables=False,  # replaces with_locals
+    max_request_body_size="always",  # replaces request_bodies
+    profiles_sample_rate=1.0,  # replaces _experiments["profiles_sample_rate"]
+    profiler_mode="thread",  # replaces _experiments["profiler_mode"]
+)
+```
+
+## Python version support
```

Comment:
```suggestion
## Python Version Support
```

Reply:
I would also move this section up to before config options since it affects whether users should migrate or not.